src/main/java/org/mockito/plugins/MockMaker.java

```diff
@@ -45,6 +46,15 @@
  * <p>Note that if several <code>mockito-extensions/org.mockito.plugins.MockMaker</code> files exists in the classpath
  * Mockito will only use the first returned by the standard {@link ClassLoader#getResource} mechanism.
  *
+ * <h3>Using the MockSettings of individual mocks</h3>
+ *
+ * <p>If you want to use your {@code AwesomeMockMaker} only for a specific mock,
+ * you can specify it using {@link MockSettings#mockMaker(String)}.</p>
+ * <pre>
+ *     Object mock = Mockito.mock(Object.class, Mockito.withSettings()
+ *             .mockMaker(AwesomeMockMaker.class));
```

Comment:
Nit: this should be updated to use the String. Let's include two examples here:

```java
 * <pre>
 *     // Use a mockmaker shipping with Mockito
 *     Object mock = Mockito.mock(Object.class, Mockito.withSettings()
 *             .mockMaker(MockMakers.INLINE));
 *     // Or load a mockmaker using a fully qualified class name
 *     Object mock = Mockito.mock(Object.class, Mockito.withSettings()
 *             .mockMaker("com.company.mockito.MyAwesomeMockMaker"));
 * </pre>
```

Reply:
Fixed.